easybuild/tools/environment.py

```diff
@@ -84,10 +84,11 @@ def setvar(key, value):
     put key in the environment with value
     tracks added keys until write_changes has been called
     """
+    oldval = os.environ.get(key, 'None (not defined)')
```

Comment:
This is just for the debug message below; it's a vain attempt to make it clear that the environment variable is not defined. But I'll do a better job here, will fix.